### PR TITLE
Update README.md Clarify how to configure rules, add links

### PR DESCRIPTION
**Pull request by hickford**
Clarify how to configure rules, add links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This repository contains an implementation of the StyleCop rules using the .NET
 The preferable way to use the analyzers is to add the nuget package [StyleCop.Analyzers](http://www.nuget.org/packages/StyleCop.Analyzers/)
 to the project where you want to enforce StyleCop rules.
 
-Currently the only way to configure the rules is to change the severity for them in a ruleset file.
+Currently the only way to configure the rules is to change the severity for them in a [ruleset file](https://msdn.microsoft.com/en-us/library/dd264996.aspx). `Settings.StyleCop` is [not supported](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/422).
```
Comment (sharwell) [inline README.md]: :bulb: File names are generally in **bold**, not `code`.

**Comment by sharwell** [comment]
How do you feel about this wording?

> The severity of individual rules may be configured using [rule set files] in Visual Studio 2015. **Settings.StyleCop** is [not currently supported].

**Comment by hickford** [comment]
Cool. Edit as you like.

**Comment by sharwell** [comment]
Thanks @hickford for taking the time to find those links and submit this pull request. They will definitely be helpful for new users. :smile:

**Pull request merged**